### ISSUE TITLE
Provide plain-text manual

### ISSUE THREAD

**Issue by xeruf** [bcpierce00/unison#598]
I had to search around for a while to find the [pdf/html manual](https://github.com/bcpierce00/unison/tree/documentation), this should definitely be more prominently linked. Furthermore the split into two wikis is quite confusing, especially as it says "FAQ wiki" on the homepage and anything else on the homepage contains less information than the Github Readme & Wiki.
There was no "Getting Started" section anywhere which provided useful pointers, only download links everywhere which I don't care about on arch ^^

Either way, it would be good to provide an `info`, `man` or even plain-text manual, ideally generated from the `tex`/`html`.

**Comment by gdt** [comment]
In theory your point is valid, and in practice I suspect no one is going to do this.

**Comment by gdt** [comment]
However, we're having issues with the current manual toolchain, and markdown or texinfo might be a reasonable approach.  Certainly if we change, a plain-text-ish format should be an important consideration.

**Comment by bcpierce00** [comment]
What's the problem with hevea?

**Comment by gdt** [comment]
See #613; not clear what it's a problem with.   There's also #558.   There is also #545, where the bug is that lynx  is needed (vs just not documented).

Earlier, I had the impression that if I ran the docs build on NetBSD, I got very different output.  This time, I didn't -- the built docs had the expected diffs only.   I don't know if it worked right this time,  or if I did the build in the same way for the previous release.

However, I have no intention of digging in to this.    I would like to see the toc link issue resolved though.  Everything else is much more theoretical.
